opendebates/templates/opendebates/snippets/idea.html

```diff
@@ -1,6 +1,8 @@
 {% load i18n %}
 {% load l10n %}
+{% load cache %}
 
+{% cache 30 question idea.id %}
```

Comment:
I was going to suggest adding `is_related` to the dependencies, but the `{% if` that looks at it has the same content on both branches, so we might as well remove the `if` entirely and just put in `col-md-3`.